From 9dd82073cfd40ddf4bcb22bec7f3ecf04bcdb0d2 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 11 Jun 2015 07:48:40 +0000
Subject: [PATCH 0591/1000] MANTIS 36825 setCORBAServants 1.0.0.00 - Mantis
 36825: setCORBAServants: description of configuration values CORBA.Servants -
 Fix configuration item's description.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@865 90b65887-3827-0410-9a23-83215b262276
---
 .../cpp/Controller/inc/setCORBASERVANTS_def.h | 37 ++++++++++++++++---
 .../cpp/Controller/src/setCORBASERVANTS.cpp   | 17 +++++----
 2 files changed, 42 insertions(+), 12 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/setCORBASERVANTS_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/setCORBASERVANTS_def.h
index 28e85c55..b78a511e 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/setCORBASERVANTS_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/setCORBASERVANTS_def.h
@@ -6,16 +6,43 @@
  ***********************************************************************
 **/
 
-#define FILE_VER_BIN     0,0,0,0
-#define FILE_VER_STR     "FileVersion", "0.0.0.0\0"
+#define FILE_VER_BIN     1,0,0,00
+#define FILE_VER_STR     "FileVersion", "1.0.0.00\0"
+
+
+/*!
+\file
+\brief Release Notes - mstarter.exe
+
+\page mstarter.exe Release Notes - mstarter.exe - process manager
+
+@version 1.0.0.00
+
+\section change1 Change history
+
+setCORBASERVANTS.exe
+
+\version 0.0.0.0
+\li 14.08.2012
+\li change - JK Mantis 24166: First implementation
+
+\version 1.0.0.00
+\li 11.06.2015
+\li change JK
+\li - Mantis 36825: setCORBAServants: description of configuration
+\li   values CORBA.Servants
+\li - Fix configuration item's description.
+
+*/
+
 
 #define IDI_ICON1 101
 #define IDI_ICON2 102
 
-#define FILEDESCRIPTION  "FileDescription"  ,"mstarter\0"
-#define INTERNALNAME     "InternalName"     ,"mstarter\0"
+#define FILEDESCRIPTION  "FileDescription"  ,"setCORBASERVANTS\0"
+#define INTERNALNAME     "InternalName"     ,"setCORBASERVANTS\0"
 #define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"mstarter.exe\0"
+#define ORIGINALFILENAME "OriginalFilename" ,"setCORBASERVANTS.exe\0"
 
 /**
  ***********************************************************************
diff --git a/SelfServiceCommon/Massai/cpp/Controller/src/setCORBASERVANTS.cpp b/SelfServiceCommon/Massai/cpp/Controller/src/setCORBASERVANTS.cpp
index 0a4e6bd0..37b9bcb7 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/src/setCORBASERVANTS.cpp
+++ b/SelfServiceCommon/Massai/cpp/Controller/src/setCORBASERVANTS.cpp
@@ -12,7 +12,7 @@
  *
  * @author Juergen Kreierhoff
  *
- * Copyright (c) 2012-2013 MATERNA Information & Communications
+ * Copyright (c) 2012-2015 MATERNA Information & Communications
  *
  **************************************************************************
 **/
@@ -62,16 +62,18 @@ int main( int argc, char ** argv )
       }
       catch( ... )
       {
+         // Item probably not yet created, try to do it now.
          try
          {
             m_config.create
                (
-                  item.c_str(),   // Config item.
-                  "rwl",           // Read-write, we must be able to
-                                  // overwrite a value.
-                  "Description: TCP/IP address and/or TCP/IP Port of CORBA-Servant to connect.SYNTAX: [host]:port",             // An explanation is very necessary.
-                  port.c_str(),   // Value
-                  32              // Max string length
+                  item.c_str(),                 // Config item.
+                  "rwl",                        // Read, write, and list
+                  "IP Port of CORBA-Servant, optional for a non-local"
+                  " servant an IP address."
+                  "\nSYNTAX: [host:]port",      // An explanation.
+                  port.c_str(),                 // Value
+                  32                            // Max string length
                );
             writeTrace
                (
@@ -81,6 +83,7 @@ int main( int argc, char ** argv )
          }
          catch( ... )
          {
+            // Oh, oh!
             writeTrace( HERE, "could not create %s", item.c_str() );
          }
       }
-- 
2.41.0.windows.1

